fastlane/Fastfile

```diff
@@ -0,0 +1,35 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+
+  before_all do
+    ensure_git_status_clean
+    update_fastlane
+  end
+
+  desc "Run test using fast lane"
+  lane :tests do 
+    run_tests(scheme: "kwarchart", 
+              devices: ["iPhone 11"])
```

Comment:
run on at least three devices, iphone se 2nd gen, iphone 8 plus and phone 12 pro max